src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -27,75 +27,62 @@ const propTypes = {
-
-                    <View style={[styles.flexGrow2]}>
-                        <SignInPageForm style={[styles.alignSelfStretch]}>
-                            <View style={[
-                                styles.componentHeightLarge,
-                                ...(props.isSmallScreenWidth ? [styles.mb2] : [styles.mt6, styles.mb5]),
-                            ]}
-                            >
-                                <ExpensifyCashLogo
-                                    width={variables.componentSizeLarge}
-                                    height={variables.componentSizeLarge}
-                                />
+                <View style={[styles.flexGrow2]}>
+                    <SignInPageForm style={[styles.alignSelfStretch]}>
+                        <View style={[
+                            styles.componentHeightLarge,
+                            ...(props.isSmallScreenWidth ? [styles.mb2] : [styles.mt6, styles.mb5]),
+                        ]}
+                        >
+                            <ExpensifyCashLogo
+                                width={variables.componentSizeLarge}
+                                height={variables.componentSizeLarge}
+                            />
+                        </View>
+                        {props.shouldShowWelcomeText && (
+                            <View style={[styles.signInPageWelcomeTextContainer]}>
+                                <Text style={[styles.mv5, styles.textLabel, styles.h3]}>
+                                    {props.welcomeText}
+                                </Text>
                             </View>
-                            {props.shouldShowWelcomeText && (
-                                <View style={[styles.signInPageWelcomeTextContainer]}>
-                                    <Text style={[styles.mv5, styles.textLabel, styles.h3]}>
-                                        {props.welcomeText}
-                                    </Text>
-                                </View>
-                            )}
-                            {props.children}
-                        </SignInPageForm>
-                    </View>
-                    <View style={[styles.mv5]}>
-                        <TermsAndLicenses />
-                    </View>
-                </KeyboardAvoidingView>
-            </View>
-        </TouchableWithoutFeedback>
-    );
-};
+                        )}
+                        {props.children}
+                    </SignInPageForm>
+                </View>
+                <View style={[styles.mv5]}>
+                    <TermsAndLicenses />
+                </View>
+            </KeyboardAvoidingView>
+        </View>
+    </TouchableDismissKeyboard>
+);
 
 SignInPageContent.propTypes = propTypes;
 SignInPageContent.displayName = 'SignInPageContent';
 
 export default compose(
     withWindowDimensions,
     withLocalize,
-
```

Comment:
@tgolen I'm getting rid of the KeyboardState HOC here since we're not using any of its props anymore, is this still needed elsewhere or is this kosher?

Reply:
I think that should be OK